SafeguardSessions/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,92 @@
+
+This error indicates issues with the SPS appliance. Try the suggestions written in the [Troubleshooting] section.
+
+### Error 10000: There is a field value missing from one or more of your filter fields.
+
+If you provide a filter field, make sure you also provide a valid filter value for it. To get help with valid values for search fields, the **List of available search queries** section in the **One Identity Safeguard for Privileged Sessions Administration Guide** on the [Technical documents for One Identity Safeguard for Priviledged Sessions] page may help.
+
+### Error 10001: Your version of the connector (ConnectorVersion) is not compatible with your SPS version (SPSVersion). For a connector version that is compatible with your SPS version, visit the official release page of the connector: https://github.com/OneIdentity/SafeguardPowerBI/releases.
+
+Revisit the [Upgrading the Power BI Connector] section.
+
+### Error 10002: The source IP returned a response with missing fields.
+
+A response returned by SPS does not contain a field that the Power BI Connector requires for processing data. Reproduce the error with trace logging enabled, and create a technical case, as described in the [Troubleshooting] section. Attach the mashup trace logs to the issue.
+
+### Error 10003: The list contains other types than text.
+
+This error can occur while the Power BI Connector transforms list values from REST response to single field by concatenating the its values. The Power BI Connector can concatenate lists containing only text values. This can indicate that SPS returned a response that is not in align with the schema. Reproduce the error with trace logging enabled, and create a technical case, as described in the [Troubleshooting] section. Attach the mashup trace logs to the issue.
+
+### Error 10004: An error happened when applying schema.
+
+In order to display the session records from SPS in a user-friendly way in Power BI, schema must be mapped to the session records. This error means that at least one session occurred in the SPS response that the schema application could not handle correctly and caused an unexpected error. Reproduce the error with trace logging enabled, and create a technical case, as described in the [Troubleshooting] section. Attach the mashup trace logs to the issue.
+
+### Error 99999: <...>
+
+This error wraps unexpected errors raised by the Power BI Connector. Reproduce the error with trace logging enabled, and create a technical case, as described in the [Troubleshooting] section. Attach the mashup trace logs to the issue.
+
+<!-- Links -->
+
+[Troubleshooting]: #troubleshooting
+
+[Installation and usage]: README.md#installation-and-usage
+[Upgrading the Power BI Connector]: README.md#upgrading-the-power-bi-connector
+
+[Technical documents for One Identity Safeguard for Priviledged Sessions]: https://support.oneidentity.com/one-identity-safeguard-for-privileged-sessions/technical-documents

```

Comment:
```suggestion
[Technical documents for One Identity Safeguard for Privileged Sessions]: https://support.oneidentity.com/one-identity-safeguard-for-privileged-sessions/technical-documents
```
also in
https://github.com/OneIdentity/SafeguardPowerBI/blob/c1ab3d3a2bb65d9719e4e77e8bc881c4f9184ac7/SafeguardSessions/TROUBLESHOOTING.md?plain=1#L34
and
https://github.com/OneIdentity/SafeguardPowerBI/blob/c1ab3d3a2bb65d9719e4e77e8bc881c4f9184ac7/SafeguardSessions/TROUBLESHOOTING.md?plain=1#L58

Reply:
Thanks! Done.